app/population/index.js

```diff
@@ -1,6 +1,8 @@
 const router = require('express').Router()
 const dailyRouter = require('express').Router({ mergeParams: true })
+const wizard = require('hmpo-form-wizard')
```

Comment:
I did a [recent fix](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/pull/1022) which replaced calls to the form wizard to allow us to support unique wizards per _thing_ in different tabs.

It might be better to use the [internal wizard call](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/07b8b4dc197b790a985f63944b08de81eb5b1087/app/allocation/index.js#L4) similar to [other app loaders](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/07b8b4dc197b790a985f63944b08de81eb5b1087/app/allocation/index.js#L31). It allows you to specify a request param to create a unique wizard. I think this is something that would be useful in this instance too.

Reply:
Update this to use the `unique-form-wizard`.

Unfortunately there isn't a unique single parameter to use - the location and date are params, and are used when editing to load a population by `id`. Would this work with being a `population.id`? Also, on create, there is no population to load, so there's no request param that can be used - how would this work in that case?

Reply:
Ah, just seen this comment after leaving one on this file again.

Yes, I see the issue with your route, you don't have a single property as you're using a combination of date and location ID. You could create a property in the middleware that sets those that you can then use in this instance. It could even just combine the two:

```javascript
# middleware
req.wizardKey = `${req.locationId}-${req.date}`

# index loader
dailyRouter.use('/edit', wizard(editSteps, editFields, editConfig, 'wizardKey'))
```

Reply:
*nods* I'd thought about using a composite key, hadn't seen any other usage of it in the system though - I've gone with your suggestion :)